src/UIAutomation/UiaOperationAbstraction/UiaOperationAbstraction.h

```diff
@@ -71,6 +71,20 @@ namespace UiaOperationAbstraction
         const char* what() const override { return "RemoteOperationsWrapper local loop break"; }
     };
 
+    class UiaFailure
+    {
+    public:
+        UiaFailure(const winrt::Microsoft::UI::UIAutomation::AutomationRemoteOperation& remoteOperation, const bool& useRemoteApi) :

```

Comment:
**nit:** It can just be `const bool`

Reply:
Or is there a reason we're only storing a reference to the bool? I think it's technically fine in this case, but seems unnecessary.

Reply:
Same with `m_remoteOperation`.